src/app/user/page.tsx

```diff
@@ -23,13 +26,24 @@ const MyHoldings = ({ showBuilderButton = false }: MyHoldingsProps) => (
 const TabsListWithButton = withBuilderButton(TabsList)
 
 export default function User() {
+  const { setMessage } = useAlertContext()
   const { address } = useAccount()
+  const { data: gauge, error: gaugeError } = useGetBuilderToGauge(address!)
 
-  const { data: isWhitelistedBuilder } = useGetIsWhitelistedBuilderV2(address!)
+  useEffect(() => {
+    if (gaugeError) {
+      setMessage({
+        severity: 'error',
+        title: 'Error loading gauge',
+        content: gaugeError.message,
+      })
+      console.error('🐛 gaugeError:', gaugeError)
+    }
+  }, [gaugeError, setMessage])
 
   return (
     <MainContainer>
-      {isWhitelistedBuilder ? (
+      {gauge && gauge !== zeroAddress ? (
```

Comment:
shouldn't this use `isBuilderOperational` instead?

Reply:
I think we should show that section even if the Builder was paused or no longer whitelisted, so checking if the gauge exists is the only way for now 👍🏼